sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -34,11 +34,13 @@ class RocketClassifier(BaseClassifier):
         The number of kernels the for Rocket transform.
     rocket_transform : str, default="rocket"
         The type of Rocket transformer to use.
-        Valid inputs = ["rocket","minirocket","multirocket"]
-    max_dilations_per_kernel : int, default=32
+        Valid inputs = ["rocket", "minirocket", "multirocket"]
+    max_dilations_per_kernel : int, optional, default=32
         MiniRocket and MultiRocket only. The maximum number of dilations per kernel.
-    n_features_per_kernel : int, default=4
+    n_features_per_kernel : int, optional, default=4
         MultiRocket only. The number of features per kernel.
+    use_multivariate : bool, optional, default=True
+        whether to use multivariate rocket transforms (True) or univariate ones (False)
```

Comment:
This one is minor, but what makes the bottom two parameters optional and the first not?

Reply:
nothing, just an oversight. All with defaults should have "optional" in docstr

Reply:
fixed